nibabel/xmlutils.py

```diff
@@ -32,7 +32,7 @@ def to_xml(self, enc='utf-8', **kwargs) -> bytes:
             Additional keyword arguments to :func:`xml.etree.ElementTree.tostring`.
         """
         ele = self._to_xml_element()
-        return b'' if ele is None else tostring(ele, enc, **kwargs)
+        return tostring(ele, enc, **kwargs)
```

Comment:
As far as I can see, `_to_xml_element` does nor return None, it raises exceptions. Besides it is declared to return `-> Element`, not `None`.

Reply:
Yeah, with type-checks this runtime check seems less necessary.